### DISCUSSION TITLE
Session Timeouts

### DISCUSSION THREAD

**Discussion by Cattrah** [YetiForceCompany/YetiForceCRM#15859] [How to] [answered]
Where can I change the length of a user session? I changed .user.ini session.gc_maxlifetime to 3600 but when we access yetiforce from a mobile device, the user is still logged out due to inactivity earlier than 60 minutes. When accessing from a computer, the session is still alive. Another acceptable solution is a Remember Me checkbox for users to click, is that a feature of yetiforce?

**Comment by rskrzypczak** [comment, marked as answer]
Here:
https://github.com/YetiForceCompany/YetiForceCRM/blob/b65343dacf96cb3fad3dc3b5c2ffdf6a220c08b7/config/Security.php#L89-L90
Remember Me checkbox: None

**Reply by Cattrah** [reply]
Thanks! That worked perfectly!
